dev-db/firebird/firebird-2.5.3.26780.0-r2.ebuild

```diff
@@ -48,7 +48,7 @@ pkg_setup() {
 	enewuser firebird 450 /bin/bash /usr/$(get_libdir)/firebird firebird
```

Comment:
This already turned out in postgresql. You shouldn't use /bin/bash here because some minimal systems don't have /bin/bash installed.